qanta/wikipedia/wikification.py

```diff
@@ -12,6 +12,7 @@ def main():
     parser.add_argument("--output_directory", type=str,
                         default="output/wikifier/data/input/",
                         help="Where we write output file")
+    parser.add_argument("--database", type=int, default=1, help="1 to load questions, 0 for no")
```

Comment:
This was also fixed in master, the parameter is deleted since it isn't needed anymore